src/main/kotlin/apply/domain/applicant/Applicant.kt

```diff
@@ -0,0 +1,37 @@
+package apply.domain.applicant
+
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+
+@Entity
+class Applicant(
+    @Column(nullable = false)
+    var name: String,
```

Comment:
기본적으로 `val` 키워드를 사용하여 모든 변수를 읽기 전용으로 선언하고 필요할 때만 `var`로 변경하면 어떨까요?

```suggestion
    val name: String,
```

Reply:
수정을 염두해두고 했던 부분인데 지금에서는 val이 더 나은 것 같네요!